readthedocs/core/permissions.py

```diff
@@ -10,9 +9,29 @@
 class AdminPermissionBase:
 
     @classmethod
-    def admins(cls, obj):
+    def owners(cls, obj):
+        """
+        Return the owners of `obj`.
+
+        If organizations are enabled,
+        we return the owners of the project organization instead.
```

Comment:
Why did you consider owners of the project to be the `OrganizationOwners` instead of the users that have ADMIN permissions over that project? I mean, members of admin Teams that include this project.

Reply:
Admins don't have all permissions to change settings from the org level. `contact_users` can be called manually from the django admin if we need to contact other type of users.

Reply:
I understand. I'm talking at a higher level, not for this particular case to contact users, but for the usage of `AdminPermission.owner` in general in other contexts as well.

With this implementation `AdminPermission.owners(project)` and `AdminPermission.owners(organization)` both return the same set of users and I think that's wrong. To me, `AdminPermission.owners(project)` should return users that have admin permissions in that particular project since we are asking for the "owners of a project" and being an owner of a project does not require to have permissions to change settings at the organization level.

In fact, thinking more about this `AdminPermission.owners` does exactly the same as the method we already had: `AdminPermission.admin`. So, we should probably delete `.owners` and just `.admins` instead of passing `project` or `organization` depending on the case required.